model/DataAccessLayer.go

```diff
@@ -208,14 +209,15 @@ func StrTrim(str string, arg []reflect.Value) (reflect.Value, error) {
 
 // StrIn will check the string instance if its equals one of the arguments, if no argument specified it will return false
 func StrIn(str string, arg []reflect.Value) (reflect.Value, error) {
-	if arg != nil && len(arg) != 0 {
+	if arg != nil && len(arg) == 0 {
```

Comment:
I don't think we actually need this if statement, since if the arg slice is nil or has length of 0 the for loop will never execute and it will fall down to the return false statement below.

Reply:
Yes agreed, for loop takes care of nil and zero len. Done the change